Backend in use: py
cdf index: 0
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[10], line 21
     19 symbols = np.array([ (0 & 0xff) | (1 << 8), (0 & 0xff) | (2 << 8) ], dtype=np.int16)
     20 enc.encode_y(symbols, idx)
---> 21 enc.flush()
     22 stream = enc.get_encoded_stream()
     23 print("Encoded stream len:", len(stream))

File ~/dcvc/DCVC_fresh/src/python/codec_loader.py:26, in RansEncoder.flush(self)
     25 def flush(self):
---> 26     self._lib.flush()

File ~/dcvc/DCVC_fresh/src/python/mlcodec_rans_py.py:123, in RansEncoderLib.flush(self)
    121         cdf_idx = combined & 0xff
    122         s = combined >> 8
--> 123         start, rng = rans_sym_grp[cdf_idx]
    124         r = RansEncPut(r, out, start, rng)
    125 elif task[0] == "z":

ValueError: too many values to unpack (expected 2)
